opacclient/opacapp/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,8 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
Are you sure we need this? It wouldn't be a killer, but since @johan12345 specifically wrote
> We think the best way for this is using Android's Storage access framework, because it does not require additional permissions and already provides you with a file chooser dialog

I want to double check.

Reply:
Yep, the permission is unnecessary considering the fact that we do not need Android <4.4 devices to have a file in the first place.